src/vaults/SyncRequestManager.sol

```diff
@@ -282,6 +250,15 @@ contract SyncRequestManager is BaseInvestmentManager, ISyncRequestManager {
         priceData.assetPerShare = PricingLib.priceAssetPerShare(priceData.poolPerShare, priceData.poolPerAsset);
     }
 
+    /// @inheritdoc IVaultManager
+    function vaultKind(IBaseVault vault_) public view returns (VaultKind, address) {
```

Comment:
QUESTION

IMO this methid should in general just live in the vault itself... Much easier and possible now as we have the adapter...

Reply:
Im not sure if that works @mustermeiszer because our SyncDepositVaults are actually 2 vaults: A SyncDepositVault and a BaseAsyncRedeemVault.

So each of these vaults doesnt know about the other one but the request manager does.